keyboards/lazydesigners/bolt/info.json

```
@@ -0,0 +1,59 @@
+    "url": "http://lazydesigners.cn",
+    "maintainer": "LAZYDESIGNERS",
+    "width": 14.25,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"K000 (F0,F1)", "x":0, "y":0},
+                {"label":"K001 (F0,F4)", "x":1, "y":0},
+                {"label":"K002 (F0,F5)", "x":2, "y":0},
+                {"label":"K003 (F0,F6)", "x":3, "y":0},
+                {"label":"K004 (F0,F7)", "x":4, "y":0},
+                {"label":"K005 (F0,C6)", "x":5, "y":0},
+                {"label":"K006 (F0,B3)", "x":7.5, "y":0},
+                {"label":"K007 (F0,B7)", "x":8.5, "y":0},
+                {"label":"K008 (F0,D0)", "x":9.5, "y":0},
+                {"label":"K009 (F0,D3)", "x":10.5, "y":0},
+                {"label":"K010 (F0,D2)", "x":11.5, "y":0},
+                {"label":"K011 (F0,D1)", "x":12.5, "y":0, "w":1.75},
+                {"label":"K100 (C7,F1)", "x":0, "y":1, "w":1.25},
+                {"label":"K101 (C7,F4)", "x":1, "y":1},
+                {"label":"K102 (C7,F5)", "x":2, "y":1},
+                {"label":"K103 (C7,F6)", "x":3, "y":1},
+                {"label":"K104 (C7,F7)", "x":4, "y":1},
+                {"label":"K105 (C7,C6)", "x":5, "y":1},
+                {"label":"K106 (C7,B3)", "x":7.75, "y":1},
+                {"label":"K107 (C7,B7)", "x":8.75, "y":1},
+                {"label":"K108 (C7,D0)", "x":9.75, "y":1},
+                {"label":"K109 (C7,D3)", "x":10.75, "y":1},
+                {"label":"K110 (C7,D2)", "x":11.75, "y":1},
+                {"label":"K111 (C7,D1)", "x":12.75, "y":1, "w":1.5},
+                {"label":"K200 (B6,F1)", "x":0, "y":2, "w":1.75},
+                {"label":"K201 (B6,F4)", "x":1.75, "y":2},
+                {"label":"K202 (B6,F5)", "x":2.75, "y":2},
+                {"label":"K203 (B6,F6)", "x":3.75, "y":2},
+                {"label":"K204 (B6,F7)", "x":4.75, "y":2},
+                {"label":"K205 (B6,C6)", "x":5.75, "y":2},
+                {"label":"K206 (B6,B3)", "x":7.25, "y":2},
+                {"label":"K207 (B6,B7)", "x":8.25, "y":2},
+                {"label":"K208 (B6,D0)", "x":9.25, "y":2},
+                {"label":"K209 (B6,D3)", "x":10.25, "y":2},
+                {"label":"K210 (B6,D2)", "x":11.25, "y":2},
+                {"label":"K211 (B6,D1)", "x":12.25, "y":2},
+                {"label":"K300 (D5,F1)", "x":0, "y":3, "w":1.25},
+                {"label":"K301 (D5,F4)", "x":1.25, "y":3, "w":1.25},
+                {"label":"K303 (D5,F6)", "x":3.25, "y":3, "w":1.25},
+                {"label":"K304 (D5,F7)", "x":4.5, "y":3, "w":2.25},
+                {"label":"K306 (D5,B3)", "x":7.25, "y":3, "w":2.25},
+                {"label":"K307 (D5,B7)", "x":9.5, "y":3},
+                {"label":"K308 (D5,D0)", "x":11.25, "y":3},
+                {"label":"K309 (D5,D3)", "x":12.25, "y":3},
+                {"label":"K310 (D5,D2)", "x":13.25, "y":3},
+                {"label":"K311 (D5,D1)", "x":13.25, "y":2}
```

Moves the `MO(2)` key object's place in the sequence.

```suggestion
                {"label":"K311 (D5,D1)", "x":13.25, "y":2},
                {"label":"K300 (D5,F1)", "x":0, "y":3, "w":1.25},
                {"label":"K301 (D5,F4)", "x":1.25, "y":3, "w":1.25},
                {"label":"K303 (D5,F6)", "x":3.25, "y":3, "w":1.25},
                {"label":"K304 (D5,F7)", "x":4.5, "y":3, "w":2.25},
                {"label":"K306 (D5,B3)", "x":7.25, "y":3, "w":2.25},
                {"label":"K307 (D5,B7)", "x":9.5, "y":3},
                {"label":"K308 (D5,D0)", "x":11.25, "y":3},
                {"label":"K309 (D5,D3)", "x":12.25, "y":3},
                {"label":"K310 (D5,D2)", "x":13.25, "y":3}
```